test/wasm_worker/wait_async.c

```diff
@@ -17,7 +17,6 @@ void worker_main()
   emscripten_wasm_notify((int32_t*)&addr, 1);
 }
 
-char stack[1024];
```

Comment:
You could also use `__builtin_wasm_tls_size() + 1024` here .. to avoid malloc, if you wanted?

Reply:
Yeah,  malloc seemed simpler. I guess the only advantage to the builtin would be that we reduce the amount of the library that the test covers?